resources/client/animations/animation.service.ts

```diff
@@ -0,0 +1,194 @@
+import { newPhoneProp, removePhoneProp } from '../functions';
+import { Delay } from '../../utils/fivem';
+
+export enum AnimationState {
```

Comment:
Maybe add these to `typings` for consistency?

Reply:
I left it out of typings mainly due to the fact that everything in typings is client & server, while this is only client.

Plus a file for a single enum isn't needed (IMO)